src/main/java/competition/subsystems/lights/LightsCommunicationSubsystem.java

```diff
@@ -109,7 +112,9 @@ public void periodic() {
         // Figure out what we want to send to the arduino
         if (!dsEnabled) {
             currentState = LightsStateMessage.RobotDisabled;
-        } else if (collector.getGamePieceCollected()) {
+        }else if(arm.isMaintainerAtGoal()){
```

Comment:
I think this logic needs to get a little more complicated. I think we want to know that the arm is at goal AND that that goal is about scoring. Maybe we can check to see if the x/z coordinates are in front of the robot as a rough proxy?

Reply:
Talked with Alex, you could check that target X position is > 12